spec/support/dtmf_helper.rb

```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+
+#
+# Convert a simple DTMF string from "1 star 2" to "dtmf-1 dtmf-star dtmf-2".
+#
+# @param [Array] sequence A set of DTMF keys, such as `%w(1 star 2)`.
+#
+# @return [String] A string with "dtmf-" prefixed for each DTMF element.
+#                  Example: "dtmf-1 dtmf-star dtmf-2".
+#
+def dtmf_seq(sequence)
+  sequence.map { |d| "dtmf-#{d}" }.join " "
```

Comment:
luz @ hound.  You changed your policy recently, didn't you? ;)

![image](https://cloud.githubusercontent.com/assets/208529/21734348/98f4437c-d428-11e6-8cf4-01bc1356aa60.png)